src/go/k8s/controllers/redpanda/console_controller.go

```diff
@@ -118,13 +118,15 @@ func (r *ConsoleReconciler) Reconcile(
 	var s state
 	switch {
 	case !console.GetDeletionTimestamp().IsZero():
+		r.Log.Info("console is in state deleting", "console", console.Name)
 		s = &Deleting{r}
 	case !console.GenerationMatchesObserved():
 		if err := r.handleSpecChange(ctx, console); err != nil {
 			return ctrl.Result{}, fmt.Errorf("handle spec change (lastObserved: %d, currentGeneration: %d): %w", console.Status.ObservedGeneration, console.GetGeneration(), err)
 		}
 		fallthrough
 	default:
+		r.Log.Info("console is in state reconciling", "console", console.Name)
```

Comment:
This will be hit too many times. This can stay as is but the level should be increased to debug or the metrics from controller runtime should be used to gather the same value.

Reply:
```suggestion
		r.Log.Info("console is in state reconciling")
```